.github/ISSUE_TEMPLATE/spike.md

```diff
@@ -0,0 +1,30 @@
+---
+name: Spike
+about: Research and/or Analysis Required
+title: ''
+labels: ":zap: spike"
+assignees: ''
+
+---
+
+# Why: Problem description :grey_question:
+A clear and concise description of what the problem is. 
+- [ ] Describe the matter that needs investigating
+- [ ] Provide context for posterity
+
+# What: Describe what a solution should resolve :grey_exclamation:  
+A clear and concise description of what a working solution should resolve, and any required deliverables (i.e. issue cards, prototypes, POCs etc.)
```

Comment:
Let's take an example, the challenge protocol v0

The intention here is to elicit the issue author to write something like:
> The expected sequence of events is roughly
> 1. blah
> 2. foo
> 3. fizzbuzz
> This spike should generate cards to implement this story.

right?

Reply:
This is more to define the "exit criteria" for a spike. It is not exclusive to breaking down a project into issues.
Spikes are often used to refine lower level criteria or determine implementation approach, which can lead to the creation of more cards, refining existing cards, and sometimes new epics.

For example:
In the case of choosing a ledger funding approach, "Analyze both ledger funding solutions, discuss recommendation with x and y, update notion documentation with the agreed upon solution (alternatively, create the issue cards to implement the chosen approach)"
"Provide an analysis of various 3rd party APIs to retrieve user location data. This 3rd party API should be able to define user location based on browser data, ip address, or user provided postal code."